### PR TITLE
Implement intervalCondition

### PR DESCRIPTION
**Pull request by phanpak**

### CODE REVIEW

File: src/strategies/DSL/conditions/intervalCondition.ts [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
-function intervalCondition (def: IntervalConditionArgs): PrimitiveArgs[] {
+const DEFAULT_MAX_INTERVALS = 0
+const DEFAULT_INITIAL_START_BLOCK = 0
 
-  // TODO: implement
-
-  return []
+function intervalCondition ({
+  id,
+  intervalMinBlocks,
```
Comment (mikec) [inline src/strategies/DSL/conditions/intervalCondition.ts:9]: these names should all match the docs

**Review by mikec — CHANGES_REQUESTED** [review]

**Comment by mikec** [comment]
@phanpak maybe we should also add a test in Strategies.test.ts that uses the `type: interval` condition in a strategy?

File: src/strategies/DSL/conditions/intervalCondition.ts (hunk 1 above)
Comment (phanpak) [reply src/strategies/DSL/conditions/intervalCondition.ts:9]: ah didn't see the mismatch. do you mean we should change brink-types to match the docs?

**Review by mikec — APPROVED** [review]
looks good! just merge after tests pass

**Pull request merged**